wiki/en/Running-a-Server.md

```diff
@@ -125,185 +144,174 @@ To register with one of the Directories built into the Jamulus Client, replace `
 |**Genre Classical/Folk** |`classical.jamulus.io:22524`|
 |**Genre Choral/Barbershop** |`choral.jamulus.io:22724`|
 
-You can also specify a [Directory](#3-directory) in the same way from the command line, providing the Server Address in the same format.
 
-#### Running as a Directory
+##### `-o or --serverinfo` 
 
-If you wish to run a [Directory](Running-a-Server#3-directory) please see [this guide](Directories).
+When registering a server on a Directory, this lets you display a server name and location details so that users can then search or sort by these values when connecting to Directories with their client.
 
-### Maintenance
+This option takes the format: 
 
-#### Viewing The Logs
+`[name];[city];[country as two-letter ISO country code]`
 
-Jamulus will log to the system log file if you left the `StandardOutput=journal` setting in the unit file.
+See [two-letter ISO country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements)
 
-To view the log, use `journalctl` (to exit press Ctrl-C). For example, to read the system log file, filtered for the Jamulus service:
+Note that semicolon and newline characters are not allowed in `name` and `city` values.
 
-`journalctl -f -u jamulus-headless`
+##### `-L or --licence` 
+Show an agreement window before users can connect
 
-#### Controlling Recording
+##### `-w or --welcomemessage` 
+Welcome message on connect. Can be given as a string or filename, and can contain HTML.
 
-When using the recording function with the `-R` command line option, if the Server receives a SIGUSR1 signal during a recording, it will start a new recording in a new Directory. SIGUSR2 will recording enabled on/off.
+##### `--serverpublicip` 
+The public IP address of the Server if connecting to a Directory behind the same NAT. See [the Directories guide](Directories) for further information.
 
-To send these signals using systemd, create the following two `.service` files in `/etc/systemd/system`, calling them something appropriate (e.g. `jamulusTogglerec.service`).
+##### `--directoryfile` 
```

Comment:
Should this be here now?  Move to [the Directories guide](Directories)?

Reply:
At last a structural comment! :-) I thought about moving these to the Directories guide but then thought maybe if all the options were on one page then people could scan them to get an overview before diving in (a bit like you do with a man page). They also turn up the TOC as convenent list.  

So it's more about the "usability" of the page. I don't think page length is a problem per se.

Reply:
If you want to write a manpage, do that.  Decide what this document represents.  I don't agree it should list the `--help` output.

Reply:
I certainly don't want to write a manpage (for an audience of less than 5% of our users, I'll leave that to others if they want to keep that updated). And I've explained in the summary to this PR, I have decided what this document represents  :-) 

"Admin" is intended to mean "how to install, set up and run a server". So I don't mean this page should list the `--help`, I mean that there are typically two stages of use for a help document: the first is where you are new to the system in question, so you want to scan through to see what it can do. The second is where you are familiar with it so you want to go straight to something you want to read about (eg recording). This is why I mention the links in my summary notes, for example (you may have missed my summary notes).

Reply:
A continuum of user types run servers. Some of them don't know they're admins. Could we banish the words _admin_ and _administration_? For example, this page title might be _Running a Server_. (I want us to banish '-ing' in titling publication-wide, but for now we have that convention.)

This page aggregates conceptual, procedural, and reference content into one page. I support this pattern up to a point, as it can aid discovery and maintenance. I also accept that reference might be aggregated into 4 buckets: _Server mode-related options_, _General Server options_, _Recording_, and _Other options_. I think this page should be two pages, _Server Guide_ and _Server Reference_, or one page in those two sections. I'd like to present CLI reference sections consistently (and I'm [cheering for table with style tweaks](https://github.com/jamulussoftware/jamuluswebsite/pull/877#issuecomment-1336112230)), with all reference content appearing in the same section (last on the page), or on its own page. I find it reassuring as an operator when I can find an at-a-glance CLI reference that aspires to be comprehensive.

Reply:
No problem with "Running a Server" as a title. 

I you want to re-structure it into Server Guide and Server Reference, then it would be good to know why you think that's better than my (or the current) take.  I've structured it according to **usage**, as opposed to content type because I wanted to avoid having to repeat content (or have readers bounce from one place to the other) in order that they can put together all the info set up, say, an Unregistered Server or Recording.

But all that would be for a separate PR. Not for this one.